cloud-provider-aws.yaml

```diff
@@ -0,0 +1,67 @@
+package:
+  name: cloud-provider-aws
```

Comment:
this package needs to be version streamed across supported kubernetes versions. We can keep the package and file name as `cloud-provider-aws-1.31` here.